replica/replica.go

```diff
@@ -776,7 +778,13 @@ func (r *Replica) createNewHead(oldHead, parent, created string) (types.DiffDisk
 	}
 
 	if _, err := os.Stat(r.diskPath(newHeadName)); err == nil {
-		return nil, disk{}, fmt.Errorf("%s already exists", newHeadName)
+		logrus.Warningf("Head file: %v already exists", newHeadName)
+		if r.getDiskSize(newHeadName) > 0 {
```

Comment:
good one..
and in linkDisk also, should we have similar check?

Reply:
in linkDisk, there should be some checks for metafile existence and delete them as well?

Reply:
No @vishnuitta if file exists in linkdisk then that is a problem, file should not exist in the first place.